style.css

```diff
@@ -1 +1,149 @@
 /*  Add your styling here */
+*{
+    margin: 0%;
+    padding: 0%;
+}
+@import url('https://fonts.googleapis.com/css2?family=Frank+Ruhl+Libre:wght@300&family=Nanum+Gothic&family=Sarabun:wght@100&display=swap');
+
+body{
+    /* width: 390px;
+    height: 844px; */
+    
+}
+.box{
+    padding: 29px;
```

Comment:
Padding 29px is quite big, to be honest! Because it means you are forcing your content too much. Padding should not be an element that can be used to adjust the content in your container, but rather use for alignments.